data/fm-info.json

```diff
@@ -3,91 +3,31 @@
-                },
-                {
-                    "memberHandle": "drmarkovi",
-                    "memberId": 982,
-                    "inducted": null,
-                    "directScore": 1575,
-                    "referrer": null
-                },
-                {
-                    "memberHandle": "cheomsk",
-                    "memberId": 552,
-                    "inducted": {"fmId":7,"inductedScoringPeriodId":5,"inductedDate":"2021-04-23","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/2.png?raw=true"},
-                    "directScore": 1350,
-                    "referrer": null
-                },
-                {
-                    "memberHandle": "crypted",
-                    "memberId": 959,
-                    "inducted": null,
-                    "directScore": 1350,
-                    "referrer": {"memberHandle":"seainvestor","memberId":684,"inducted":null,"referralScore":[0,21,5,18,15,270,0]}
-                },
-                {
-                    "memberHandle": "okayko",
-                    "memberId": 605,
-                    "inducted": null,
-                    "directScore": 750,
-                    "referrer": {"memberHandle":"lopegor","memberId":1369,"inducted":{"fmId":8,"inductedScoringPeriodId":5,"inductedDate":"2021-04-23","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/39.png?raw=true"},"referralScore":[0,100,25,51,71,150,80]}
-                },
-                {
-                    "memberHandle": "igrex",
-                    "memberId": 1048,
-                    "inducted": null,
-                    "directScore": 650,
-                    "referrer": {"memberHandle":"leet_joy","memberId":957,"inducted":null,"referralScore":[0,35,98,130,80,130,85]}
-                },
-                {
-                    "memberHandle": "vagif",
-                    "memberId": 1747,
-                    "inducted": null,
-                    "directScore": 590,
-                    "referrer": {"memberHandle":"slavik_investor","memberId":1642,"inducted":null,"referralScore":[0,50,0,0,15,118,0]}
-                },
-                {
-                    "memberHandle": "leopold",
-                    "memberId": 1295,
-                    "inducted": null,
-                    "directScore": 450,
-                    "referrer": null
-                }
+                {"memberHandle":"tomato","memberId":2,"inducted":{"fmId":1,"inductedScoringPeriodId":0,"inductedDate":"2021-02-21","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/31.png?raw=true"},"directScore":4500,"referrer":null},
+                {"memberHandle":"andybut","memberId":1316,"inducted":{"fmId":9,"inductedScoringPeriodId":7,"inductedDate":"2021-06-07","avatar":"https://github.com/Joystream/founding-members/blob/main/avatars/primary-avatar/1337.png?raw=true"},"directScore":3000,"referrer":null},
```

Comment:
This members image is not being rendered, probably because 1337.png was a placeholder (I don't see it in the avatars folder)?

Reply:
You're right, I forgot to mention it has two placeholders!
Me having submitted a summary as well :)

Reply:
Fixed the one you pointed out, will have to let the other stay until @blrhc can review!

@DzhideX, if this one renders, please merge ASAP, as we're 4min overdue...